plugins.txt

```diff
@@ -0,0 +1,18 @@
+ansible
+blueocean
+blueocean-i18n
+blueocean-config
+blueocean-events
+blueocean-web
+blueocean-commons
```

Comment:
These plugins are for Blue Ocean Jenkins and are not specific to any one project. These plugins are in aid of 'pipeline as code' and all project will benefit and are likely to want to use pipeline editor.
For further info, please see this link: https://jenkins.io/projects/blueocean/